src/game/models.ts

```diff
@@ -1,39 +1,52 @@
 import type { BasePlayer } from '@player/models';
+import type { BoardProps } from '@shared/Board/models';
+import type { Boat } from '@src/boat/models';
 import type { GamePageParams } from '@pages/game/[gameId]';
 import type { Weapon } from '@weapon/models';
 
+export interface WaitingForTheHostProps {
+  className?: string;
+}
+
 export type GamePageContentProps = Required<GamePageParams>;
 
 export type GameView = 'boats_placement' | 'playing' | 'settings';
 
 export interface GameSettings {
-  boardDimensions: number;
+  boardDimensions: BoardProps['dimensions'];
   weapons: Weapon['name'][];
```

Comment:
Maybe this property could be name weaponNames in order to fit the type ?
```suggestion
  weaponNames: Weapon['name'][];
```

Reply:
✅